### ISSUE TITLE
Scheduler Keyboard Navigation improvements

### ISSUE THREAD

**Issue by kirchoni** [telerik/kendo-react#1060]
Currently, we're applying `tabIndex=0` only when an item or a slot is selected. This allows us to easily tab into the element when returning the focus to the component.

However, initial navigation to the item/slot is not available without additional application code, as shown in the following stackblitz example:
https://stackblitz.com/edit/react-anxhiq?file=app/main.tsx

We can further enhance the keyboard capabilities of the scheduler by applying a default `tabIndex` to `0` on the first available item, or to the first available slot in cases where no items are present in the current view.

**Comment by amang95** [comment]
Do we have an ETA on this? Is this going to be a part of the next release?

**Comment by kspeyanski** [comment]
The issue has been fixed in version [4.9.0](https://www.telerik.com/kendo-react-ui/components/changelogs/ui-for-react/) of the `@progress/kendo-react-scheduler` package.

**Comment by amang95** [comment]
Thank you for fixing this. But when I use the keyboard to navigate and press enter to open a popup for creating events or editing events, I am not able to either navigate around the popup form or even close it using the keyboard.
Does that need another issue to be created?
Or is there a workaround?

**Comment by amang95** [comment]
Also, the view-more link below the events in the month view, cannot be brought into focus using keyboard navigation while tabbing.
